packages/ui/src/components/PropertiesModal/PropertiesModal.tsx

```diff
@@ -0,0 +1,57 @@
+import { Modal } from '@patternfly/react-core';
+import { Caption, Table, Tbody, Thead } from '@patternfly/react-table';
+import { FunctionComponent, useState } from 'react';
+import { CamelComponentTableHeaders, CamelComponentTableRows, KameletTableHeaders, KameletTableRows } from '.';
```

Comment:
Sometimes importing from the adjacent barrel file makes `jest` a bit "uncomfortable", better to import the desired file directly and avoid using `from '.';` :smiley: